### DISCUSSION TITLE
ppi_client : De donde me la bajo?

### DISCUSSION THREAD

**Discussion by unosyceros** [itAtPPI/ppi-connector-python#8] [General]
Buenas, estoy intentando ver los ejemplos subido pero no encuentra la ppi_cliente, saben de donde puedo obtenerla?

gracias.

**Comment by exql** [comment]
Hola, buenas!!
Probá descargandola de este link: https://pypi.org/project/ppi-client/
Yo descargué ese paquete y funcionó
